.github/workflows/shellcheck.yml

```
@@ -0,0 +1,17 @@
+name: shellcheck
+
+# Only run tests on unprotected branches.
+# Otherwise we waste build minutes re-running tests will know will pass.
+on:
+  push:
+    branches:
+      - '**'
+      - '!develop'
+      - '!main'
```

This is a public repository, so minutes are free.

```suggestion
on: [push]
```